tests/statetrie_iterator.go

```diff
@@ -0,0 +1,58 @@
+package main
```

Comment:
You should add copyrights on the top of the code.

Reply:
Sorry some late.
This code has been changed to verify-tools.